src/server/plugins/engine/models/FormModel.ts

```diff
@@ -118,9 +114,20 @@ export class FormModel {
     this.fieldsForContext = new ComponentCollection(exposedComponentDefs, this)
     this.fieldsForPrePopulation = {}
     this.pages = def.pages.map((pageDef) => this.makePage(pageDef))
+
+    // All models get an Application Status page
+    this.pages.push(
```

Comment:
Let's discuss how we add the `/status` page.
Pushing in onto the `pages` stack here, but eventually it might be a page we get the user to define (as we do for summary).

(BTW, this is originally how the summary page worked - it was fully automated and the user didn't need to define it)